### ISSUE TITLE
Create conformance file with multi layer image

### ISSUE THREAD

**Issue by cconcolato** [AOMediaCodec/av1-avif#40]

**Comment by cconcolato** [comment]
Discussed in the Still Picture WG. Today aomenc cannot produce multi-layered streams, but we are confident that the use of multi-streams in AVIF is still possible. Bitstreams will be produced when a multi-layer encoder is available.

**Comment by cconcolato** [comment]
Jai will check if some at Google can produce a simple multi-layer video stream from which we would extract a multi-layer key frame.

**Comment by cconcolato** [comment]
Thanks. I had a quick look at `av1-1-b8-22-svc-L2T1.ivf`. Indeed, the first temporal unit contains 2 frames: a key frame and an inter frame. This first temporal unit could be used for making a image and the whole sequence to create a image sequence. One thing surprises me: the OBU sequence header indicates a resolution of 1280x720 while the 2 frames indicate a resolution of 640x360. Is the sequence meant to be exercising spatial scalability or quality scalability?

**Comment by wantehchang** [comment]
Cyril: The av1-1-b8-22-svc-L2T1.ivf test video was contributed to libaom by Soo-Chul Han of Vidyo in this libaom changelist:

  https://aomedia-review.googlesource.com/c/aom/+/73081

I will email your question to Soo-Chul Han and cc you.

**Comment by schan86** [comment]
This is Soo-Chul.  av1-1-b8-22-svc-L2T1.ivf is indeed a bitstream that is exercising SPATIAL scalability.  The name L2T1 implies "2 spatial layers and 1 temporal layer"

**Comment by cconcolato** [comment]
Can you clarify why the frames in both layers have the same resolution?

**Comment by cconcolato** [comment]
This was discusssed in today's meeting and it seems the file above is not produced by libaom but downloaded. We need volunteers to modify an existing encoder to produce such files.

**Comment by cconcolato** [comment]
Here is a an example file: [av1-1-b8-22-svc-L2T1.avif.zip](https://github.com/AOMediaCodec/av1-avif/files/5235145/av1-1-b8-22-svc-L2T1.avif.zip)
It may not be entirely correct but it contains an image with 2 layers.
Also, it is very simple, no `lsel` property.

**Comment by cconcolato** [comment]
If it helps, here is MP4Box's XML dump of the item payload:
```
<OBU OperatingPointIdc="0" bit_depth="8" color_description_present_flag="0" color_primaries="2" color_range="0" has_ext="0" has_size_field="1" header_size="2" height="720" level="1" matrix_coefficients="2" profile="0" size="15" spatialID="0" still_picture="0" temporalID="0" transfer_characteristics="2" type="seq_header" width="1280"/>
<OBU frame_type="key" has_ext="0" has_size_field="1" header_size="2" refresh_frame_flags="255" show_existing_frame="0" show_frame="1" size="27" spatialID="0" temporalID="0" type="frame_header" uncompressed_header_bytes="25"/>
<OBU has_ext="0" has_size_field="1" header_size="3" nb_tiles="1" size="13162" spatialID="0" temporalID="0" type="tile_group">
  <Tile number="0" size="13158" start="4"/>
</OBU>
<OBU has_ext="0" has_size_field="1" header_size="4" nb_tiles="1" size="24495" spatialID="0" temporalID="0" type="tile_group">
  <Tile number="0" size="24490" start="5"/>
</OBU>
<OBU has_ext="0" has_size_field="1" header_size="3" nb_tiles="1" size="2971" spatialID="0" temporalID="0" type="tile_group">
  <Tile number="0" size="2967" start="4"/>
</OBU>
<OBU frame_type="inter" has_ext="1" has_size_field="1" header_size="3" refresh_frame_flags="8" show_existing_frame="0" show_frame="1" size="31" spatialID="1" temporalID="0" type="frame_header" uncompressed_header_bytes="28"/>
<OBU has_ext="1" has_size_field="1" header_size="5" nb_tiles="2" size="17454" spatialID="1" temporalID="0" type="tile_group">
  <Tile number="0" size="3013" start="10"/>
  <Tile number="1" size="14431" start="3023"/>
</OBU>
<OBU has_ext="1" has_size_field="1" header_size="4" nb_tiles="2" size="6081" spatialID="1" temporalID="0" type="tile_group">
  <Tile number="0" size="5970" start="9"/>
  <Tile number="1" size="102" start="5979"/>
</OBU>
```
Note that the image is tiled. It doesn't have to be and FrameOBUs could have been used instead of TileGroup OBUs.

**Comment by leo-barnes** [comment]
Per the last meeting, this will be resolved by #131 and #134.
